### PR TITLE
IA-2063: Register, add button to create a children

### PR DESCRIPTION
**Pull request by beygorghor**
Add a button to create a new org unit as the children of the current one.
Related JIRA tickets : IA-2063

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [x] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

- accept levels as default param for parent while creating a org unit
- fetch parent info end set it as parent to the new org unit
- custom paper component for registry
- add link to create children org unit with current org unit as level in the url
- placeholder on map while loading list of children

## How to test

Go to register for an existing org unit.
You should see a + button nearby the name of the org unit, if you click on it you should arrive on the org unit creation page with the org unit selected in registry as parent 

## Print screen / video

https://user-images.githubusercontent.com/12494624/233610010-6f303b2b-9d17-4000-ad6a-3e0259e85455.mov

## Notes

⚠️ Current base is this [PR](https://github.com/BLSQ/iaso/pull/535)

### CODE REVIEW

File: hat/assets/js/apps/Iaso/domains/app/translations/en.json [hunk 1]
```diff
@@ -773,6 +773,7 @@
     "iaso.queryBuilder.supportedTypeFields": "Supported type of fields",
     "iaso.queryBuilder.title": "Query builder",
     "iaso.queryBuilder.xlsQuestionsTypesLink": "Possible questions types",
+    "iaso.registry.addOrgUnitChildren": "Add children org unit",
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/app/translations/en.json]: shouldn't it be "Add new child org unit"?

File: hat/assets/js/apps/Iaso/domains/registry/components/OrgUnitPaper.tsx
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/registry/components/OrgUnitPaper.tsx:1]: Can we not refactor `WidgetPaper` to take a list of buttons instead?

**Review by quang-le — CHANGES_REQUESTED** [review]
Just check that EN trabnslation, and maybe spin a ticket off for the WidgetPaper

File: hat/assets/js/apps/Iaso/domains/registry/components/OrgUnitPaper.tsx
Comment (beygorghor) [reply hat/assets/js/apps/Iaso/domains/registry/components/OrgUnitPaper.tsx:1]: nope, wodget paper is having also expand collapse behaviour and icon...

File: hat/assets/js/apps/Iaso/domains/app/translations/en.json [hunk 2]
```diff
@@ -773,7 +773,7 @@
     "iaso.queryBuilder.supportedTypeFields": "Supported type of fields",
     "iaso.queryBuilder.title": "Query builder",
     "iaso.queryBuilder.xlsQuestionsTypesLink": "Possible questions types",
-    "iaso.registry.addOrgUnitChildren": "Add children org unit",
+    "iaso.registry.addOrgUnitChildren": "Add new children org unit",
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/app/translations/en.json]: child, singular 😆

**Review by quang-le — CHANGES_REQUESTED** [review]

**Review by quang-le — APPROVED** [review]

**Pull request merged**